Formula/osc-cli.rb

```diff
@@ -0,0 +1,26 @@
+class OscCli < Formula
+  include Language::Python::Virtualenv
+
+  desc "Official Outscale CLI providing connectors to Outscale API"
+  homepage "https://github.com/outscale/osc-cli"
+  url "https://github.com/outscale/osc-cli/releases/download/v1.6.0/osc-sdk-1.6.tar.gz"
+  sha256 "6319af4ee3f2effd991e8bd0fcd2b49914cf200915a7b1f606957db8eeb14ed8"
```

Comment:
This works better with our automation
```suggestion
  url "https://files.pythonhosted.org/packages/8c/29/cf68365132a0ac906278abdc704569ce8f1ba4c609a2067cd00c5ed22271/osc-sdk-1.6.tar.gz"
  sha256 "fb312c82c73ed4eba7bbb3a028e4d2f53e00e28c576ff2f638258f27a855e7e7"
```